manager/testdata/new-blueprints/read-module.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: app.m4d.ibm.com/v1alpha2
+kind: M4DModule
+metadata:
+  name: arrow-flight-common-read-conf
+  namespace: m4d-system
+  labels:
+    version: 0.0.1
+spec:
+  description: "This module knows how to configure a cluster wide arrow flight service"
+  # this field is true if this module serves multiple users. Otherwise it
+  # is false. It is used for example when setting the endpoint to
+  # access the module service - if it is a multi-user module then
+  # the service name will be fixed and will be taken from
+  # M4DModule.Capabilities.Api.endpoint.host.
```

Comment:
I'll add the endpoint into the status of the blueprint. (It's already in the plotter status. I just did not come along to also update the blueprints) 
I think returning possible different endpoints for every time an application runs is not user friendly. This is also why @tomersolomon1 added the possibility to specify an endpoint that the user expects.

What would be nice for the user:
1. User specifies one endpoint (service name) in the same namespace as the application.
2. M4D manages this service and makes sure that it points to the correct module
3. In the best case ALL data access goes via arrow-flight through this endpoint. (Thus all the user does is change the asset name and a different asset will be retrieved)

Reply:
Are you assuming that only multi-user services (one per cluster) will have an endpoint? Otherwise, like in the current read scenarios, multiple services are deployed in the same cluster+namespace, thus manager calculates the endpoints given some module-defined prefix/suffix plus the release name in order to distinguish between different releases of the same service.